StoreApp/Views/Store/DeleteProduct.cshtml

```diff
@@ -0,0 +1,102 @@
+﻿@model StoreApp.Models.Store.DetailsProductViewModel
```

Comment:
Need to discuss this form. I'm not sure that we need to have a form which is used to removing a product. It would be good if you show me the working application

Reply:
I deleted this model